src/wp-includes/wp-db.php

```diff
@@ -2551,24 +2549,22 @@ protected function process_fields( $table, $data, $format ) {
 				}
 			}
 
+			wp_load_translations_early();
+
 			if ( 1 === count( $problem_fields ) ) {
-				if ( function_exists( '__' ) ) {
+				$this->last_error = sprintf(
 					/* translators: %s Database field where the error occurred. */
```

Comment:
Let's add a colon here, for consistency with other translator comments:
```suggestion
					/* translators: %s: Database field where the error occurred. */
```